Add Power Support ppc64le and python version

nagesh4193 opened the pull request:
Added recent python version and power support for the travis.yml file with ppc64le. This is part of the Ubuntu distribution for ppc64le. This helps us simplify testing later when distributions are re-building and re-releasing.

zcutlip commented on .travis.yml:
```diff
@@ -1,10 +1,13 @@
 language: python
 python:
   - "2.7"
-  - "3.4"
```
Is there a reason we're dropping python 3.4?

zcutlip commented:
Thanks for the patch. Should be easy to merge. Please see comment re: python 3.4

nagesh4193 commented:
on Amd64 Architecture, for the command pip install ., we see the below error.

PyYAML requires Python '!=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*' but the running Python is 3.4.8

since 3.4 is not supported and it is older version, I have removed 3.4 version and to maintain the number of versions for testing, I have added 3.8 recent python version. Hopefully this clarifies about 3.4 version removal. Thank you.
